src/quo2/components/settings/accounts/view.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.settings.accounts.view
+  (:require [quo.react-native :as rn]
+            [quo2.components.avatars.account-avatar :as av]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.settings.accounts.style :as style]))
+
+(defn card-background [{:keys [custom-color]}]
+  [:<>
+   [rn/view {:style (style/background-top custom-color)}]
+   [rn/view {:style (style/background-bottom)}]])
+
+(defn avatar [{:keys [_color _icon] :as avatar-props}]
+  [rn/view {:style (style/avatar-border)}
+   [av/account-avatar (assoc avatar-props :size 48)]])
+
+(defn menu-button [{:keys [on-press]}]
+  [rn/view {:style style/menu-button-container}
+   [button/button {:style    (style/menu-button-color)
+                   :type     :gray
+                   :icon     true
+                   :size     24
+                   :on-press on-press}
+    :i/more]])
+
+(defn account
+  [{:keys [account-name account-address avatar-icon custom-color on-press-menu]}]
+  [rn/view {:style style/card}
+   [card-background {:custom-color custom-color}]
+   [rn/view {:style style/card-top}
+    ;; Avatar
```

Comment:
I agree,
I deleted all of these comments, since it's clear what the UI is defining below them.

I added 57acac2ed0769bbf7f8936a600e9c6327a703ace & 425e7ae51662824569fb1f53faa5ceecf8e11f07 with the changes.